src/app/application.cpp

```diff
@@ -908,12 +912,11 @@ int Application::exec()
                     , QString::number(m_webui->port()));
             printf("%s\n", qUtf8Printable(tr("To control qBittorrent, access the WebUI at: %1").arg(url)));
 
-            const Preferences *pref = Preferences::instance();
-            if (pref->getWebUIPassword() == QByteArrayLiteral("ARQ77eY1NUZaQsuDHbIMCA==:0WMRkYTUWVT9wVvdDtHAjU9b3b7uB8NR1Gur2hmQCvCDpm39Q+PsJRJPaCU51dEiz+dTzh8qbPsL8WkFljQYFQ=="))
+            if (const QString tempPassword = m_webui->tempPassword(); !tempPassword.isEmpty())
             {
-                const QString warning = tr("The Web UI administrator username is: %1").arg(pref->getWebUiUsername()) + u'\n'
-                        + tr("The Web UI administrator password has not been changed from the default: %1").arg(u"adminadmin"_s) + u'\n'
-                        + tr("This is a security risk, please change your password in program preferences.") + u'\n';
+                const QString warning = tr("The Web UI administrator username is: %1").arg(m_webui->username()) + u'\n'
+                        + tr("The Web UI administrator password was not set. You can use a temporary password:: %1").arg(tempPassword) + u'\n'
+                        + tr("You are encouraged to set your own password in program preferences.") + u'\n';
```

Comment:
>You should update it to your own password in program preferences.

I think it is incorrect to use "update" in this case. Do not forget that the password has not yet been set by the user, so that there is something to update.

Reply:
Then: `You should change it to your own password in program preferences.`